docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - POSTGRES_USER=${DATABASE_USERNAME}
       - POSTGRES_DB=${DATABASE}
     ports:
-      - "5432:5432"
```

Comment:
Found that my postgres runs on 5432, so i can't set the docker to 5432, as it is already taken, but i always change this when i'm resolving conflicts.